go.mod

```diff
@@ -4,8 +4,8 @@ go 1.18
 
 require (
 	github.com/IBM-Cloud/bluemix-go v0.0.0-20230601050310-eecebfbff63e
-	github.com/IBM-Cloud/container-services-go-sdk v0.0.0-20230118060037-101bda076037
-	github.com/IBM-Cloud/power-go-client v1.2.4
+	github.com/IBM-Cloud/container-services-go-sdk v0.0.0-20230822142550-30562e113de9
+	github.com/IBM-Cloud/power-go-client v1.2.2
```

Comment:
You are rolling back the changed of this PR
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4766/files

Reply:
updated